src/search/SearchMessagesScreen.js

```diff
@@ -78,33 +62,46 @@ class SearchMessagesScreen extends PureComponent<Props, State> {
   };
 
   handleQueryChangeInner = async (id: number, query: string) => {
-    let messages: Message[];
-    {
-      // if the promise's construction fails, we let the exception
-      // propagate immediately
-      const networkPromise = this.performQueryRaw(query);
-
-      try {
-        messages = await networkPromise;
-      } finally {
-        /* Succeed or fail, we update our request-state. We discard late
+    const { dispatch } = this.props;
+    let apiResponse;
+    const fetchArgs = {
+      narrow: SEARCH_NARROW(query),
+      anchor: LAST_MESSAGE_ANCHOR,
+      numBefore: 20,
+      numAfter: 0,
+    };
+
+    // if the promise's construction fails, we let the exception
+    // propagate immediately
```

Comment:
This comment is explaining something that's no longer true after this change. It's about how the `performQueryRaw` call is made outside the `try`, and only the `await` is inside.

I think in fact it's fine, and clearer, to put them together as you do in this commit :-) See https://github.com/gnprice/zulip-mobile/commit/db8fbd16e2c0c96b3a128b861e61369ffe9f60fa (part of #3911) which makes the same change. The comment should be deleted to match, though.

Reply:
Deleted the comment.